app/models/stream_events/classification_event.rb

```diff
@@ -13,6 +13,11 @@ def process
 
       cache_linked_models!
 
+      if linked_workflow["nero_config"].keys.include? "webhooks"
```

Comment:
This should probably be in `Workflow.update_cache` (called by `cache_linked_models!` below).